STYLE_GUIDE.md

```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  Write hash params in contentxt dependent way.
```

Comment:
> contentxt dependent 

what is `contentxt dependent way`? This is ok for change log but not for style guide (it should understand the guy who doesn't know about rubocop)